File: AdvantageFramework.Core/Registry/Attributes/RegistryAttribute.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace AdvantageFramework.Core.Registry.Attributes
{
    [AttributeUsage(AttributeTargets.Field)]
    public partial class RegistryAttribute : Attribute
    {

        /* TODO ERROR: Skipped RegionDirectiveTrivia */


        /* TODO ERROR: Skipped EndRegionDirectiveTrivia */
        /* TODO ERROR: Skipped RegionDirectiveTrivia */


        /* TODO ERROR: Skipped EndRegionDirectiveTrivia */
        /* TODO ERROR: Skipped RegionDirectiveTrivia */
        private string _Section = "";
        private string _Key = "";
        private string _Default = "";

        /* TODO ERROR: Skipped EndRegionDirectiveTrivia */
        /* TODO ERROR: Skipped RegionDirectiveTrivia */
        public string Section
        {
            get
            {
                string SectionRet = default;
                SectionRet = _Section;
                return SectionRet;
            }

            set
            {
                _Section = value;
            }
        }

        public string Key
        {
            get
            {
                string KeyRet = default;
                KeyRet = _Key;
                return KeyRet;
            }

            set
            {
                _Key = value;
            }
        }

        public string Default
        {
            get
            {
                string DefaultRet = default;
                DefaultRet = _Default;
                return DefaultRet;
            }

            set
            {
                _Default = value;
            }
        }

        /* TODO ERROR: Skipped EndRegionDirectiveTrivia */
        /* TODO ERROR: Skipped RegionDirectiveTrivia */
        public RegistryAttribute(string Section, string Key, string Default)
        {
            _Section = Section;
            _Key = Key;
            _Default = Default;
        }

        /* TODO ERROR: Skipped EndRegionDirectiveTrivia */
    }
}

File: AdvantageFramework/Registry/Attributes/RegistryAttribute.vb
Namespace Registry.Attributes

    <AttributeUsage(AttributeTargets.Field)> _
    Public Class RegistryAttribute
        Inherits Attribute

#Region " Constants "



#End Region

#Region " Enum "



#End Region

#Region " Variables "

        Private _Section As String = ""
        Private _Key As String = ""
        Private _Default As String = ""

#End Region

#Region " Properties "

        Public Property Section() As String
            Get
                Section = _Section
            End Get
            Set(ByVal value As String)
                _Section = value
            End Set
        End Property
        Public Property Key() As String
            Get
                Key = _Key
            End Get
            Set(ByVal value As String)
                _Key = value
            End Set
        End Property
        Public Property [Default]() As String
            Get
                [Default] = _Default
            End Get
            Set(ByVal value As String)
                _Default = value
            End Set
        End Property

#End Region

#Region " Methods "

        Public Sub New(ByVal Section As String, ByVal Key As String, ByVal [Default] As String)

            _Section = Section
            _Key = Key
            _Default = [Default]

        End Sub

#End Region

    End Class

End Namespace